testing/kubernetes/bookinfo.yaml

```diff
@@ -31,7 +31,7 @@ spec:
 ---
 
 apiVersion: v1
-kind: ReplicationController
+kind: ReplicaSet
```

Comment:
The `ReplicaSet` kind is still an extension API object, so you have to use `apiVersion: extensions/v1beta1`. See [here](https://kubernetes.io/docs/user-guide/replicasets/).